languagetool-language-modules/en/src/test/java/org/languagetool/rules/en/EnglishRepeatedWordsRuleTest.java

```diff
@@ -49,6 +49,7 @@ public void testRule() throws IOException {
     assertCorrectText("This is a new experience. Happy New Year!");
     assertCorrectText("This was needed. There is a need to do it.");
     assertCorrectText("It needs to be done. That also needed to be done.");
+    assertCorrectText("I still need to sign-in somewhere. You need to sign-in too.");
```

Comment:
I think it should be "sign in", just as a side note 🤓

Reply:
With one of the solutions in the other comment, this sentence should be correct, too:

```
assertCorrectText("I still needed to sign-in somewhere. She needs to sign-in too.");
```

Reply:
Perhaps?
I'll drop the hyphen in 'sign-in' once you drop the hyphen in 'extra-careful' XD
What even is that emoji...